independent-publisher-connectors/RestCountries/apiDefinition.swagger.json

```diff
@@ -0,0 +1,795 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "REST Countries",
+        "description": "Get information about countries via a RESTful API",
+        "version": "1.0",
+        "contact": {
+            "name": "Siddharth Vaghasia",
+            "email": "siddh.vaghasia@gmail.com"
+        }
+    },
+    "host": "restcountries.com",
+    "basePath": "/v3.1/",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/all": {
+            "get": {
+                "responses": {
+                    "200": {
+                        "description": "200 OK",
```

Comment:
@siddharth-vaghasia Can it be just "OK", instead of "200 OK"?

Reply:
yes...I have updated this..